cirq/google/api/v2/program.proto

```diff
@@ -99,6 +111,16 @@ message Operation {
 
   // Which qubits the operation acts on.
   repeated Qubit qubits = 3;
+
+  // Token that can be used to specify a version of a gate.
+  // For instance, a gate that has been calibrated for a circuit.
+  //
+  // The token can be specified as a string or as a reference to
+  // the constant table of the circuit.
+  oneof arg {
```

Comment:
Maybe call the overall oneof field "token" instead? Also might be nice to use a common prefix on the fields to make clear they relate to the token:
```
oneof token {
  string token_value = 4;
  int32 token_constant_index = 5;
}
```

Reply:
Done.